build/helper/metadata_add_all.py

```diff
@@ -291,13 +291,34 @@ def _add_python_name(a, attributes):
     attributes[a]['python_name'] = n
 
 
+def _add_converter_defaults(a, attributes):
+    '''Add 'python_api_converter_name_to_driver' and 'python_api_converter_name_from_driver' as None if they do not exist'''
+    if 'python_api_converter_name_to_driver' not in attributes[a]:
+        attributes[a]['python_api_converter_name_to_driver'] = None
+    else:
+        attributes[a]['python_api_converter_name_to_driver'] = '_converters.' + attributes[a]['python_api_converter_name_to_driver']
+
+    if 'python_api_converter_name_from_driver' not in attributes[a]:
+        attributes[a]['python_api_converter_name_from_driver'] = None
+    else:
+        attributes[a]['python_api_converter_name_from_driver'] = '_converters.' + attributes[a]['python_api_converter_name_from_driver']
+
+
+def _add_default_attribute_class(a, attributes):
+    '''Set 'attribute_class' if not set'''
```

Comment:
Expand on what 'attribute_class' is, and/or add to the metadata wiki (that thing needs to be looked over)

Reply:
Expanded docstring